### PR TITLE
feat: Enable creating redirects in Drupal and generate HTML redirects in Gatsby.

### PR DESCRIPTION
**Pull request by miromichalicka**

### CODE REVIEW

File: drupal/composer.lock [hunk 1]
```diff
@@ -2609,9 +2609,6 @@
                         "[web-root]/profiles/README.txt": "assets/scaffold/files/profiles.README.txt",
                         "[web-root]/themes/README.txt": "assets/scaffold/files/themes.README.txt"
                     }
-                },
```
Comment (miromichalicka) [inline drupal/composer.lock:2612]: @petrillek don't we need this patch anymore? It's not in composer.json and therefore is automatically removed from composer.lock

File: gatsby/config/create-pages.ts [hunk 2]
```diff
@@ -245,4 +252,11 @@ export const createPages: GatsbyNode['createPages'] = async ({
       });
     });
   });
+
+  // Create redirects
+  const redirects = result.data.allRedirect.nodes;
```
Comment (miromichalicka) [inline gatsby/config/create-pages.ts:257]: How to protect this from failing if no redirects exists?

Comment (kanei) [reply gatsby/config/create-pages.ts:257]: `const redirects = typeof result.data.allRedirect.nodes !== 'undefined' ? result.data.allRedirect.nodes : [];` this way?

Comment (miromichalicka) [reply gatsby/config/create-pages.ts:257]: it will fail on query already

**Review by vitkolos — APPROVED** [review]

**Pull request merged**